### ISSUE TITLE
What happens with multiple-level path?

### ISSUE THREAD

**Issue by sbeaudoin** [MicrosoftDocs/winrt-api#496]
It seems the method only support a single sub-folder name and fails with a "Incorrect Parameter" if we try to create otherwise, for example "levelA\levelB\levelC".  Please, gives precision for this behavior and an alternative if there is one.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f986c1bb-d68a-ba82-99f7-7264f254de37
* Version Independent ID: a85cf9da-f6ae-1c67-6612-adeea44f8e8f
* Content: [StorageFolder.CreateFolderAsync Method (Windows.Storage) - UWP app developer](https://docs.microsoft.com/en-us/uwp/api/windows.storage.storagefolder.createfolderasync#Windows_Storage_StorageFolder_CreateFolderAsync_System_String_)
* Content Source: [https://cpubwin.visualstudio.com/DefaultCollection/windows-uwp/_git/winrt-api-build?path=%2fwinrt-api-build%2fwindows.storage.storagefolder.yml&version=GBlive&_a=contents](https://cpubwin.visualstudio.com/DefaultCollection/windows-uwp/_git/winrt-api-build?path=%2fwinrt-api-build%2fwindows.storage.storagefolder.yml&version=GBlive&_a=contents)
* Product: **windows**
* Git Login: **wdg-dev-content**
* Microsoft Alias: **wdg-dev-content**

**Comment by hkhmsft** [comment]
Thanks for submitting your feedback and I apologize for the delayed response. I see that updates have not been made since the creation of this Issue so I went ahead and created an internal work item for new content. The timeline for this may vary, so I will go ahead and close this and circle back when the remark or the code snippet has been added. 

Though the Issue will be marked as Closed, I will monitor any replies that come afterwards so treat this like an open Issue. Many thanks!
